### ISSUE TITLE
Challenge Details: Final Submission Guidelines is missing

### ISSUE THREAD

**Issue by SathyaJayabal** [topcoder-platform/community-app#4429]
Challenge Details: Final Submission Guidelines is missing in V5 version
V5 URL: https://test-community-app.topcoder-dev.com/challenges/465d9fd6-d7a8-42fb-8a33-c0070c9adaf2
V4 URL:https://qa-community-app.topcoder-dev.com/challenges/30056664  
<img width="1440" alt="Screenshot 2020-05-27 at 2 49 42 PM" src="https://user-images.githubusercontent.com/58783823/83001482-7083a380-a029-11ea-9ddd-68418c4d2edc.png">
<img width="1440" alt="Screenshot 2020-05-27 at 2 50 00 PM" src="https://user-images.githubusercontent.com/58783823/83001500-77121b00-a029-11ea-8c65-278725e55ebb.png">

**Comment by sushilshinde** [comment]
@topcoder-platform/topcodercompetitors @topcoder-platform/tcxcommunity this is open for pickup

Submit in 8 hours and PR to

https://github.com/topcoder-platform/community-app/tree/integration-v5-challenge-api
https://github.com/topcoder-platform/topcoder-react-lib/tree/integration-v5-challenge-api

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30126887 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30126887 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by rashmi73** [comment]
@sushilshinde @rootelement  we need finalSubmissionGuidelines property in api response.

1. hit https://api.topcoder-dev.com/v5/challenges/?id=2b8e84ea-656c-4ee5-bf1d-7030b50d2e9a
2. observe the response doesnt have "finalSubmissionGuidelines" property hence block is hidden on details page.
3. I see challenge-api repo needs modification to include "finalSubmissionGuidelines" property in response whereas topcode-react-lib requires no modification.

**Comment by rootelement** [comment]
No, final guideline submissions were removed in v5. When data is migrated forward into v5, Final Guideline Submissions were appended onto the challenge spec.

**Comment by SathyaJayabal** [comment]
@rootelement in that case can you check the migration script please as none of the migrated challenges have the final submission guidelines ?

cc @sushilshinde

**Comment by rootelement** [comment]
This is available in dev and is currently being updated:
http://api.topcoder-dev.com/v5/challenges/5f94881b-5d64-4c92-b43e-df5f0cf93dfe

Please give some time for it to finish.

**Comment by rashmi73** [comment]
@rootelement can I proceed with this now?

**Comment by sushilshinde** [comment]
@rashmi73 please go ahead

**Comment by rashmi73** [comment]
@sushilshinde it is completed raising PR

**Comment by sushilshinde** [comment]
Sure

**Comment by rashmi73** [comment]
@rootelement "finalSubmissionGuidelines" is missing at 
http://api.topcoder-dev.com/v5/challenges/5f94881b-5d64-4c92-b43e-df5f0cf93dfe

kindly confirm

**Comment by rashmi73** [comment]
@rootelement @sushilshinde waiting here

**Comment by SathyaJayabal** [comment]
@rashmi73 , the finalSubmissionGuidelines will be  part of contest spec. So I don't think we will need front-end  change  here.

**Comment by SathyaJayabal** [comment]
@rootelement @sushilshinde 
while some of the contests now have Final Submission Guidelines, some contests are still missing that section

example where final submission guidelines is present
https://beta-community-app.topcoder.com/challenges/0a2b2f4a-548e-4311-8714-9c28a663f25c
https://www.topcoder.com/challenges/30126528
<img width="1440" alt="Screenshot 2020-06-01 at 6 04 32 PM" src="https://user-images.githubusercontent.com/58783823/83410606-49b3db80-a434-11ea-99ed-33bcc269de1b.png">
<img width="1440" alt="Screenshot 2020-06-01 at 6 04 35 PM" src="https://user-images.githubusercontent.com/58783823/83410619-50dae980-a434-11ea-8bbc-a6a8d1be8ddf.png">

example where final submission guidelines is missing
https://beta-community-app.topcoder.com/challenges/55cff8bd-c622-412d-9683-ed85398044f7
https://www.topcoder.com/challenges/30108180
<img width="1440" alt="Screenshot 2020-06-01 at 6 08 24 PM" src="https://user-images.githubusercontent.com/58783823/83410090-57b52c80-a433-11ea-8734-c2f32d124edd.png">
<img width="1440" alt="Screenshot 2020-06-01 at 6 08 27 PM" src="https://user-images.githubusercontent.com/58783823/83410110-5d127700-a433-11ea-9605-6ee0d5d05877.png">
https://beta-community-app.topcoder.com/challenges/5ec0a02f-ffe2-4ce1-bd37-bd5a26f37ea2
https://www.topcoder.com/challenges/30126024

<img width="1440" alt="Screenshot 2020-06-01 at 6 07 34 PM" src="https://user-images.githubusercontent.com/58783823/83410117-60a5fe00-a433-11ea-911f-edbb4587a98a.png">
<img width="1440" alt="Screenshot 2020-06-01 at 6 07 36 PM" src="https://user-images.githubusercontent.com/58783823/83410125-626fc180-a433-11ea-8cd8-901b7426ed72.png">

**Comment by SathyaJayabal** [comment]
@sushilshinde , not able to verify because the above challenges have not been migrated during the latest migration
https://github.com/topcoder-platform/challenge-api/issues/149

**Comment by SathyaJayabal** [comment]
verified on beta
<img width="1440" alt="Screenshot 2020-06-10 at 9 47 42 AM" src="https://user-images.githubusercontent.com/58783823/84226544-af245e00-aaff-11ea-913b-cd106247ff1d.png">
<img width="1440" alt="Screenshot 2020-06-10 at 9 48 41 AM" src="https://user-images.githubusercontent.com/58783823/84226558-b481a880-aaff-11ea-8360-bc645e3119a7.png">
@sushilshinde , we cam close this. tcx_accepted label needed.
